src/entities/person/person.ts

```diff
@@ -961,6 +961,20 @@ export class Person extends UniverseEntity<PersonPayload, PersonRawPayload> {
       throw new PersonPreviewNotificationError(undefined, { error: err })
     }
   }
+
+  public async checkDuplicates (): Promise<PossibleDuplicatesRawPayload[]> {
```

Comment:
for that case we'd need need to define an interface for the query that allows for
```
{
  strategies: ['global_phonenumber']
}
```

which then should be passed to...